projects/plugins/protect/src/js/components/scan-footer/index.jsx

```diff
@@ -74,6 +75,11 @@ const ProductPromotion = () => {
 
 const FooterInfo = () => {
 	const { hasRequiredPlan } = useProtectData();
+	const { stats } = useWafData();
+	const { globalStats } = stats;
+	const totalVulnerabilitiesFormatted = parseInt(
+		globalStats.totalVulnerabilities
+	).toLocaleString();
```

Comment:
```suggestion
	const totalVulnerabilities = parseInt( globalStats?.totalVulnerabilities );
	const totalVulnerabilitiesFormatted = isNaN( totalVulnerabilities ) ? '50,000' : totalVulnerabilities.toLocaleString();
```

Just to be safe - handling of missing or invalid stats data with fallback value.